commit 08101ff532626926d40393bec771ebd09035c068
Author: Robin <robin.harper@sydney.edu.au>
Date:   Sat Sep 18 07:59:11 2021 +1000

    fixed incorrect index in conditionalMutualInfo

diff --git a/src/marginal.jl b/src/marginal.jl
index 2c394bd..0dc4baa 100644
--- a/src/marginal.jl
+++ b/src/marginal.jl
@@ -559,7 +559,7 @@ end
     then does the calculation, inside the sum for that x,y,z
 """
 function getSummand(x,y,z,jpXYZ,jpXZ,jpYZ,jpZ)
-    multipland = size(jpYZ)[1]
+    multipland = size(jpXZ)[1]
     pxyz = jpXYZ[multipland*(y-1)+x,z]
     if pxyz == 0
         return 0
@@ -575,6 +575,7 @@ end
 
 """
 conditionalMutualInfo
+    Pass in vectors of qubits for X Y and Z and the distribution to analyse.
 
 ## Discussion
     Gives the conditional mutual information for the qubits
